adapt to elementary genlist_item_insert_before API change from r55869

diff -uNr trunk2/src/gui.c trunk/src/gui.c
--- trunk2/src/gui.c	2010-02-01 07:56:03.000000000 +0100
+++ trunk/src/gui.c	2011-01-30 00:36:15.000000000 +0100
@@ -1067,7 +1067,7 @@
 	if(strcmp(sel_category, " All Tasks ")==0) strcpy(Task[i].cat, "Personal");
 	else strcpy(Task[i].cat, sel_category);
 	if(item) { 
-		task_list[i] = elm_genlist_item_insert_after(list, &itc1, &Task[i], item, ELM_GENLIST_ITEM_NONE,
+		task_list[i] = elm_genlist_item_insert_after(list, &itc1, &Task[i], NULL, item, ELM_GENLIST_ITEM_NONE,
 								  NULL, NULL);
 	}
 	//cater for no items in list
